AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -266,6 +266,7 @@ damota.me,pracazarohom.*,seduo.*#%#//scriptlet('remove-class', 'force--consent|s
 !
 ! NOTE: Regular rules
 !
+laroche-posay.ru#%#//scriptlet('trusted-set-cookie', 'selectedCookieGroups', '{"strictlyNecessaryCookies":"1"}') 
```

Comment:
I'd prefer non-trusted rules
```adblock
! https://github.com/AdguardTeam/AdguardFilters/issues/176447
!#if (!adguard_app_ios && !adguard_ext_android_cb)
||laroche-posay.ru/static/*/js/cookies-popup/
laroche-posay.ru#$#body { overflow: auto !important; }
!#endif
```

You can add to https://github.com/AdguardTeam/AdguardFilters/blob/85a9c5ef49dbec57790583b04213268d4fd392dd/AnnoyancesFilter/Cookies/sections/cookies_specific.txt#L11186

Reply:
Do non-trusted rules work? It worked only with a trusted rule for me

Reply:
Okay, I added them there

Reply:
```
||laroche-posay.ru/static/*/js/cookies-popup/
laroche-posay.ru#$#body { overflow: auto !important; }
```
will work also in another ad blockers. Also if it is possible to not write cookies - avoid it.

Reply:
Okay, I'll do that